src/ui/modals/confirm_queue_clear.rs

```diff
@@ -38,7 +40,7 @@ pub struct ConfirmQueueClearModal<'a> {
 impl ConfirmQueueClearModal<'_> {
     pub fn new(context: &AppContext) -> Self {
         let mut button_group_state = ButtonGroupState::default();
-        let buttons = vec![Button::default().label("Save"), Button::default().label("Cancel")];
+        let buttons = vec![Button::default().label("Confirm"), Button::default().label("Cancel")];
```

Comment:
I think it should say "Clear" instead to name a specific action instead of generic confirm.

Reply:
No objection!